docs/source/sawtooth_cli/admin_genesis.rst

```diff
@@ -0,0 +1,57 @@
+Admin Genesis CLI Tool
+======================
+
+Overview
+--------
+
+The genesis CLI tool produces a file for use during initialization of a
+validator.  A network requires an initial block (known as the genesis block)
+whose signature will determine the block chain id.  This initial block is
+produced from a list of batches, which will be applied at genesis time.  The
+input to the command is a set of zero or more files containing serialized
+``BatchList`` protobuf messages. The output is a file containing a serialized
+``GenesisData`` protobuf message.  This file, when placed at
+``<sawtooth_data>/genesis.batch``, will trigger the genesis process.
+
+The location ``sawtooth_data`` depends on whether or not the environment
+variable ``SAWTOOTH_HOME`` is set.  If it is, then ``sawtooth_data`` is located
+at ``<SAWTOOTH_HOME>/data``.  If it is not, then ``sawtooth_data`` is located
+at ``/var/lib/sawtooth``.
+
```

Comment:
Added these lines to talk about the data directory - will need to be updated once the file-based configuration of disk locations is in place.